src/main/kotlin/cryptoservice/controller/CryptoController.kt

```diff
@@ -38,9 +38,10 @@ class CryptoController(
     @GetMapping("/decrypt")
     fun decrypt(
         @RequestHeader gcpAccessToken: String,
+        @RequestHeader agePrivateKey: String,
```

Comment:
Skal vi injecte denne i crypto-service eller burde den sendes med hver gang? 😊
Skal jeg evt. fjerne det andre endepunktet som er satt opp for dekryptering med age-nøkkel?

Reply:
Hva mener du med å "sende med denne i crypto-service"? 

Funksjonen over var mer en huskelapp for nye utviklere om at det er sånn vi burde gjøre det, så tenker at det bare er å utvide dette endepunktet og fjerne `/decrypt-with-age-key` 😊

Reply:
Tror spm er om vi skal bruke age-private-key som en miljøvariabel for crypto-service vs. å sende den fra RiSc-backend hver gang man skal bruke decrypt?

Reply:
Ja, det Hans Kristian sa - skal man sende det med ved hver spørring, eller som en miljøvariabel?

Reply:
Så det var et eget endepunkt som var laget for å sende med age-nøkkel, og at det var en bedre måte å gjøre det på?

Reply:
Er agePrivateKey statisk i hele applikasjonens levetid? Eller endrer den seg fra request til request?

Om den er statisk så syns eg absolutt den bør tas bort fra requesten og gjøres om til ein miljøvariabel. Kan dette gjøre at backenden slipper å ha kjennskap til privateAgeKey?

Om den endrer seg så forstår eg ikkje heilt korleis den kan bli ein miljøvariabel.

Reply:
I vår løsning så er jo tanken at denne age-nøkkelen skal kunne brukes til å låse opp roser dersom det er noe som skjer med en bunkers-nøkkel (om ikke ting dette har endret seg de siste månedene), eller at man skal ha muligheten til å lese ros med 1. denne age-nøkkelen, og 2. age-nøkkel eid av dataprodukt-tjeneste. 😊

Reply:
Da forstår eg ikkje heilt korleis (og korfor) du ønsker å bruke denne nøkkelen som ein miljøvariabel 🧐

Kanskje eg misforstår problemstillingen? 😅

Reply:
Hmm, nå tror jeg vi snakker forbi hverandre.
1. Den er statisk, og man skal kunne bruke denne til å låse opp alle roser som er kryptert av denne servicen. Derfor skjønner ikke jeg helt hvorfor man skal ha muligheten til å sende inn forskjellige age-nøkler(Med mindre noe har endret seg de siste månedene).
2. Den brukes allerede som en miljøvariabel i cryptoservice i skip. 

Grunnen til at jeg løftet dette som et spørsmål var fordi jeg så kommentarer rundt i koden om at man burde sende med dette som et parameter istedenfor.

Reply:
Ja OK, da forstår eg 😎
99% støtter miljøvariabel.
(1% er om vi ønsker å støtte "1 crypto-service - flere virksomheter" i fremtiden 🙈)